charts/gardener/values.yaml

```diff
@@ -75,6 +75,10 @@ controller:
       #
       # for google-clouddns we need to provide the following keys:
       # serviceaccount.json: '{"type": "service_account", "project_id": "...", ...}'
+      # 
+      # for alicloud-dns we need to provide the following keys:
```

Comment:
Can't this be merged with line 72? Let's not duplicate this.

Reply:
No. For Alicloud, the naming is not the same as AWS. To track clear reference to secret which Alicloud needs, I keep the orginal name from Alicloud portal[ see https://usercenter.console.aliyun.com/#/manage/ak]